psmc_check.py

```diff
@@ -27,6 +27,8 @@
 import Ska.engarchive.fetch_sci as fetch
 from Chandra.Time import DateTime
 import Chandra.Time
+from Quaternion import Quat
```

Comment:
FYI all these emails are listed as coming from Tom Aldcroft.

Only if one does a reply will the attribution appear (see below).

The "From" ought to be who it's from. Or maybe we need to start signing 
at the end of writing the email.

Gregg


On 06/01/2016 11:26 AM, John ZuHone wrote:
>
> In psmc_check.py 
> <https://github.com/sot/psmc_check/pull/6#discussion_r65383256>:
>
> > @@ -27,6 +27,8 @@
> >  import Ska.engarchive.fetch_sci as fetch
> >  from Chandra.Time import DateTime
> >  import Chandra.Time
> > +from Quaternion import Quat
>
> I don't think we need this line, actually.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/sot/psmc_check/pull/6/files/e98688cf8880ddee6aa581da9851a5f4a6b0c4ee#r65383256>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ABpcCmKAYr9pGlzPlOyVA8byYLS-guuHks5qHaSPgaJpZM4IrqE1>.
>